library/database/class.sqldriver.php

```diff
@@ -754,8 +754,10 @@ public function getCountLike($table = '', $like = false) {
      *
      * @param string $tableName The name of the table to delete from.
      * @param array $wheres An array of where conditions.
+     * @param int $limit Limit the number of records to delete.
+     * @return string Returns an DML statement.
      */
-    public function getDelete($tableName, $wheres = []) {
+    public function getDelete($tableName, $wheres = [], $limit = 0) {
```

Comment:
Why are we adding this parameter to a method that just triggers an error?

Reply:
So that the base class and subclass match definitions.